### PR TITLE
UI

### PR DESCRIPTION
**Pull request by MW-0317**
Merging UI into main to then create a separate branch for each individual component for UI. Modified project settings to fit this repo's project settings to the best of my ability (there seems only to be differences in names, ie. dotnet assembly name, and project name). This UI branch has included LaTeX representation when adding new functions to the function palette.

### CODE REVIEW

**Comment by PROMETHIA-27** [comment]
Looks like there's a lot of stuff that should not be committed in this right now. I'll try adding to the .gitignore file and removing them from this branch when I get the chance

**Comment by MW-0317** [comment]
Added .gitignore and removed binary files that were unnecessary.

File: ui.cs
Comment (PROMETHIA-27) [inline ui.cs:1]: is this file and ui.gd necessary?

Comment (MW-0317) [reply ui.cs:1]: The .gd script is necessary as it provides functionality for the exit button, the .cs script however is not. I will push an updated version to UI.

**Review by PROMETHIA-27 — APPROVED** [review]

**Pull request merged**